manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -523,3 +519,5 @@ defaults:
 
     # Set to `type: RuntimeDefault` to use the default profile for templated gateways, if your container runtime supports it
     seccompProfile: {}
+_internal_do_not_set:
```

Comment:
This part I don't like very much, since `values.yaml` is user-facing.

So have a slight preference for un-nesting `pilot/cni` entirely via `zzz_profile.yaml` - since we have already committed to that pattern anyway, and already do similar in that spot for other things.